amazon_cloud_code_generator/data/plugins/module_utils/core.py

```diff
@@ -408,12 +403,35 @@ def delete_resource(self, type_name: str, identifier: str) -> bool:
 
         return changed
 
+    def ensure_request_status(self, response: Dict) -> bool:
+        # Wait until reource request becomes IN_PROGRESS
+        response: Dict = {}
+        wait_for_request: bool = False
+
+        for i in count():
```

Comment:
Since you aren't using `i` anywhere, you can just use `while True:` and skip the unnecessary import. I'll also add that we need some way to break out of this loop if the request status gets stuck in the pending state, otherwise this will loop forever. We should probably honor the `wait_timeout` here.

Reply:
Good catch, fixed that.